content/css/concepts/background/terms/background-image/background-image.md

```diff
@@ -32,43 +32,57 @@ Where `<image>` can be one of the following:
   - `radial-gradient(circle at top, #40BFA5, #BB40BF)`
   - `repeating-radial-gradient(#1D4D19, #191A4D 10px)`
 
-Multiple background images are supported. The first value specified will be stacked on top.
+Multiple background images are supported. The first `value` specified will be stacked on top.
 
 ```css
-background-image: <image>, <image>;
+background-image: value, value;
 ```
 
-**Note:** The `background-color` property should always be set in case the image is unavailable.
+**Note:** The `background-color` property should always be set, which acts as a fallback color when the set `background-image` is unavailable.
 
 ## Example 1
 
-Set background image on the `.hero` class element as a linear gradient starting with `yellow` at the top and `purple` at the bottom:
+Set a linear gradient background on the `.hero` class element, transitioning from yellow at the top to purple at the bottom:
 
 ```css
 .hero {
+  width: 400px;
+  height: 300px;
   background-image: linear-gradient(yellow, purple);
 }
 ```
 
+![Image illustration for example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/background-image-example-1.png)
```

Comment:
Add a more descriptive `alt-text` (text inside the square brackets). Something like this:
Linear Gradient background from yellow to purple